recipes/glog/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+autoconf
+autoreconf -i
```

Comment:
This may look a bit hacky, but it was necessary to do this because the configure file provided was broken and the first pass of `autoconf` was not enough. Without the `-i`, `autoreconf` fails to recognize one of the dependencies is going to built later.